### ISSUE TITLE
RSLHelperAdd.DLL not found!

### ISSUE THREAD

**Issue by Jinn-James** [FarbstoffRSL/RSL-Helper#289]
### Operating system

Windows 10

### Systemlanguage

English (EN)

### Current Game-Version

v5.10.3

### Issues

1. RSLHelperAdd.DLL not found
2. Message Window popup: "Wait for a new Raid Window!"
3. RSLHelperAdd.DLL DOES exist??

### RSLHelperAdd.DLL not found
![RSLHelper_no_dll](https://user-images.githubusercontent.com/60804808/151677620-d32078c1-ec36-49c6-b395-f19668ad718f.png)

### "Wait for new a Raid Window!"
![RSLHelper1](https://user-images.githubusercontent.com/60804808/151677909-dbe31847-a440-4821-87a7-5ffed32e7b8a.png)

### RSLHelperAdd.DLL DOES exist??
![Rslhelperdll_LI](https://user-images.githubusercontent.com/60804808/151678234-46c13ce2-405b-42c5-9ea4-8a7eb7f688d6.jpg)

### Issues - Details

When RSLHelper is loading an error message window says "RSLHelperAdd.DLL not found". After clicking the "Ok" button, RSLHelper starts up and seems to run ok. But then a window says, "Wait for new a Raid Window!". And then nothing else happens. Although the application. still has some functionality. Plarium Play and RaidSL both run properly with admin rights. I did a few campaign runs but to no avail. My file explorer shows that the RSLHelperAdd.DLL file is present. I have tried some of your solutions that are relevant in the RSLHelper FAQ's GitHub page. I have retried both install types from the Google Drive page.

Am I not doing something right? Is it possible to download the RSLHelperAdd.DLL file and replace the existing file? Even though it says the file does not exist.

Please help. It would be much appreciated! I'm really looking forward to using this application.
Thank you for making it!

**Comment by OpaWattebausch** [comment]
Watch your antivirus, ramsonware or other security software. It is prevented from installing on your pc.

**Comment by Jinn-James** [comment]
I completely uninstalled the program. Then lowered User Account Control from setting #3 to setting #2. Then redownloaded and installed. I don't know if this actually helped or not. But the first and second time I uninstalled/reinstalled I did not lower UAC settings. The program runs fine now. Although it's still periodically a bit glitchy. CB simulator didn't work though. But I am not concerned with that aspect of the program and will not be using it anyways.

@OpaWattebausch: Thank you for pointing me in the right direction. Your help is much appreciated.
